### DISCUSSION TITLE
[Feature Request] Send picture over Telegram on Tasmota32-webcam

### DISCUSSION THREAD

**Discussion by SHA-NDY** [arendst/Tasmota#11051] [Ideas]
It would be nice to have picture messages functionality over Telegram on tasmota32-webcam builds.
An example can be found on https://randomnerdtutorials.com/telegram-esp32-cam-photo-arduino/.

**Comment by supperka** [comment]
this will be awsome !

**Comment by nikvostro** [comment]
would be great

**Comment by 2000ede** [comment]
I'd also like to send photos to Telegram using Tasmota. Ideally, this would work for each script.
I've tried several things, but haven't been able to get it to work.

Is there any update on this?
